src/controller/postRouter.js

```diff
@@ -105,6 +106,17 @@ postRouter.get("/", async (req, res) => {
  *      200 - Reply successfully created
  *      400 - That post doesn't exist
  */
+postRouter.post("/tags", async (req, res) => {
+    try {
+        const posts = await postService.checkTags(req.body.tags, req.body.inclusive);
+        res.status(200).json({
+            Posts: posts
```

Comment:
I'm not sure if this route is getting posts with tags or creating a tag. It looks like you are returning multiple posts so if you are getting posts it should be a get not a post. You'd also have to use req.query instead of req.body if you change it to get

Reply:
Oh yeah, I was so used to post endpoints I forgot it should be get